lib/async.js

```diff
@@ -879,6 +882,8 @@
             concurrency: concurrency,
             payload: payload,
             saturated: noop,
+            unsaturated:noop,
+            buffer: concurrency / 4,
```

Comment:
the difference between the saturation point and a buffer point you want in order to say the queue is unsaturated ...

Here I'm saying once the queue has been saturated i want it to be at 25% - concurrency in order to say the queue is unsaturated..

Example 
concurrency = 10 //workers
saturated = 10 //items in queue
unsaturated = 7.5 //items in queue or less..